.github/workflows/ci.yml

```diff
@@ -195,21 +197,29 @@ jobs:
           ddev phpunit --testsuite unit
           ddev phpunit --bootstrap=./vendor/weitzman/drupal-test-traits/src/bootstrap-fast.php --configuration ./phpunit.xml --testsuite existing-site
 
-      - name: Change admin password to make it easier for Cypress tests
-        run: ddev drush upwd "$CYPRESS_ADMIN_USERNAME" "$CYPRESS_ADMIN_PASSWORD"
-
-      - name: Cypress Test
-        run: ddev cypress-run
-
-      - name: Save Cypress recordings
-        if: always()
-        uses: actions/upload-artifact@v4
-        with:
-          name: cypress-recordings
-          path: |
-            web/themes/custom/contribtracker/cypress/screenshots
-            web/themes/custom/contribtracker/cypress/videos
-          retention-days: 3
+      #- name: Change admin password to make it easier for Cypress tests
```

Comment:
Why comment out everything?

Also, let's discuss the test execution times. We still want to run _some_ tests in the regular CI flow.

Reply:
> Why comment out everything?
> 
> Also, let's discuss the test execution times. We still want to run _some_ tests in the regular CI flow.

Created a separate workflow for it.